lib/l10n/app_en.arb

```
@@ -1,5 +1,403 @@
+  "ib_home": "Interactive Book Home",
+  "loading_chapters": "Loading Chapters...",
+  "show_toc": "Show Table of Contents",
+  "@ib_page_view": {},
+  "copyright_notice": "Copyright © 2021 Contributors to CircuitVerse. Distributed under a [CC-by-sa] license.",
+  "table_of_contents": "Table of Contents",
+  "navigate_previous_page": "Tap to navigate to previous page",
+  "navigate_next_page": "Tap to navigate to next page",
+  "loading_image": "Loading image...",
+  "image_load_error": "Failed to load image",
+  "no_image_source": "No image source provided",
+  "no_image_source": "No image source provided",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)